### DISCUSSION TITLE
Can I estimate between-subject variability only for subjects where it applies?

### DISCUSSION THREAD

**Discussion by billdenney** [nlmixr2/nlmixr2#170] [Q&A] [answered]
I have a model with different between-subject variability (BSV) for the healthy vs the patient population.  Is there a way to estimate BSV only for the population of interest and not estimate BSV for the other population?

In the model, I set the parameter estimate like:

```
e0_healthy <- exp(le0_healthy + etae0_healthy)
e0_patient <- exp(le0_patient + etae0_patient)
e0 <- e0_healthy*HEALTHY + e0_patient*(1 - HEALTHY)
```

Where `le0_healthy` and `le0_patient` are fixed effects in the model, `etae0_healthy` and `etae0_patient` are random effects and `HEALTHY` is an indicator variable if the patient is healthy (1) or a patient (0).  The `ini()` block of the model is normal with no indication of healthy/patient:

```
ini({
le0_healthy <- log(1.1)
le0_patient <- log(2.1)
etae0_healthy ~ 0.1
etae0_patient ~ 0.2
})
```

(and some other stuff that's not relevant.)

Is there a better way?

**Comment by mattfidler** [comment, marked as answer]
This is the only way I can think of with mu referencing.
